src/screens/nano/NanoContractList.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useContext } from 'react';
+import $ from 'jquery';
+import { t } from 'ttag'
+import { Link } from 'react-router-dom'
+import hathorLib from '@hathor/wallet-lib';
+import ModalRegisterNanoContract from '../../components/nano/ModalRegisterNanoContract';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { NANOCONTRACT_REGISTER_STATUS } from '../../sagas/nanoContract';
+import { GlobalModalContext, MODAL_TYPES } from '../../components/GlobalModal';
+
+
+/**
+ * List of nano contracts of a blueprint
+ *
+ * @memberof Screens
+ */
+function NanoContractList() {
+  const context = useContext(GlobalModalContext);
+  const nanoContracts = useSelector(state => state.nanoContracts);
+  const navigate = useNavigate();
+
+  const registerNC = () => {
+    context.showModal(MODAL_TYPES.NANOCONTRACT_REGISTER);
+  }
+
+  const createNC = () => {
+    navigate('/nano_contract/select_blueprint/');
```

Comment:
This screen is probably part of another PR, correct?
In any case, what do you think of using `/nano_contract/create` instead?

Reply:
Yes, this is the next PR. I decided to use `select_blueprint` because there's a first screen for the user to select the blueprint before the screen to create the nano contract.

We might use this `select_blueprint` screen for other reasons, so I prefer to keep it more generic like this.